src/test/java/org/terracotta/offheapstore/AbstractOffHeapMapIT.java

```diff
@@ -74,6 +75,34 @@ public AbstractOffHeapMapIT(Generator generator) {
   protected abstract Map<Integer, byte[]> createOffHeapBufferMap(PageSource source);
 
   @Test
+  public void testRemoveByHash() throws Exception {
+    final int ENTRIES_COUNT = 1013;
+    Map<SpecialInteger, SpecialInteger> map = createMap(generator);
+    if (!(map instanceof HashingMap)) {
+      throw new AssertionError("map is instance of " + map.getClass());
```

Comment:
That's a leftover, IMHO the `map` should be blindly cast to `HashingMap` and let the test throw `ClassCastException` in case an implementation does not comply.